sdk/dotnet/Pulumi.Automation.Tests/LocalWorkspaceTests.cs

```diff
@@ -1659,12 +1659,12 @@ public async Task InlineProgramExceptionDuringUpShouldNotDeleteResources()
                 }
             });
 
-            var config = new Dictionary<string, ConfigValue>()
-            {
-                ["ShouldFail"] = new ConfigValue("false"),
-            };
             try
             {
+                var config = new Dictionary<string, ConfigValue>
+                {
+                    ["ShouldFail"] = new ConfigValue("false"),
+                };
```

Comment:
I'm curious why this is better

Reply:
Moving the `config` var changes the block scope and fixes the Resharper issue "Local variable 'config' hides outer local variable with the same name" for line 1641.